DESCRIPTION

```diff
@@ -18,4 +18,5 @@ Depends:
     rAzureBatch (>= 0.4.0)
 Suggests:
     testthat, caret, plyr, lintr
+Remotes: Azure/rAzureBatch
```

Comment:
hmm. I'm not 100% sure but it seems that yes, you do. Test it at first. Remove both azure packages and try to install on your local machine ```doAzureParallel```